packages/astro/src/types/public/config.ts

```diff
@@ -1328,58 +1328,57 @@ export interface ViteUserConfig extends OriginalViteUserConfig {
 		 * - `constrained` - The image will scale to fit the container, maintaining its aspect ratio, but will not exceed the specified dimensions.
 		 * - `fixed` - The image will maintain its original dimensions.
 		 * - `full-width` - The image will scale to fit the container, maintaining its aspect ratio.
 		 */
-		experimentalLayout?: ImageLayout | undefined;
+		layout?: ImageLayout | undefined;
 		/**
 		 * @docs
-		 * @name image.experimentalObjectFit
+		 * @name image.objectFit
 		 * @type {ImageFit}
 		 * @default `"cover"`
 		 * @description
 		 * The default object-fit value for responsive images. Can be overridden by the `fit` prop on the image component.
-		 * Requires the `experimental.responsiveImages` flag to be enabled.
+		 * Requires a value for `layout` to be set.
 		 */
-		experimentalObjectFit?: ImageFit;
+		objectFit?: ImageFit;
 		/**
 		 * @docs
-		 * @name image.experimentalObjectPosition
+		 * @name image.objectPosition
 		 * @type {string}
 		 * @default `"center"`
 		 * @description
 		 * The default object-position value for responsive images. Can be overridden by the `position` prop on the image component.
-		 * Requires the `experimental.responsiveImages` flag to be enabled.
+		 * Requires a value for `layout` to be set.
 		 */
-		experimentalObjectPosition?: string;
+		objectPosition?: string;
 		/**
 		 * @docs
-		 * @name image.experimentalBreakpoints
+		 * @name image.breakpoints
 		 * @type {number[]}
 		 * @default `[640, 750, 828, 1080, 1280, 1668, 2048, 2560] | [640, 750, 828, 960, 1080, 1280, 1668, 1920, 2048, 2560, 3200, 3840, 4480, 5120, 6016]`
 		 * @description
-		 * The breakpoints used to generate responsive images. Requires the `experimental.responsiveImages` flag to be enabled. The full list is not normally used,
+		 * The breakpoints used to generate responsive images. Requires a value for `layout` to be set. The full list is not normally used,
 		 * but is filtered according to the source and output size. The defaults used depend on whether a local or remote image service is used. For remote services
 		 * the more comprehensive list is used, because only the required sizes are generated. For local services, the list is shorter to reduce the number of images generated.
 		 */
-		experimentalBreakpoints?: number[];
+		breakpoints?: number[];
 		/**
 		 * @docs
-		 * @name image.experimentalDefaultStyles
+		 * @name image.responsiveStyles
 		 * @type {boolean}
 		 * @default `true`
 		 * @description
-		 * Whether to automatically add global styles to ensure that experimental images resize correctly. This is enabled by default, but can be disabled if you want to manage the styles yourself.
-		 * This option is only used when the `experimental.responsiveImages` flag is enabled.
+		 * Whether to automatically add global styles to ensure that responsive images resize correctly. This is enabled by default, but can be disabled if you want to manage the styles yourself.
```

Comment:
And if the above is correct, we need to update the following as well. Maybe something like:
```suggestion
		 * Whether to automatically add global styles to ensure that responsive images resize correctly. This is disabled by default, but can be enabled if you want Astro to manage the styles for you.
```

Reply:
> Whether to add Astro's default global styles for resizing responsive images. Enable this for automatic responsive images when adding the `layout` attribute to the `<Image />` component. This is disabled by default to allow you to opt-in to the additional global styles, or to allow you to control responsiveness yourself.

Yes, and... maybe the framing here is also different now. We generally WANT people to choose true, even though the default is false. So I think this might be more helpful if it's a bit more opinionated about what is happening here? (I know we're still working out the implementation details, so this is just an example of how we could take this a step further...)

Reply:
Good idea. I'll say that it's recommended.